thumbor/proc-requirements.txt

```diff
@@ -0,0 +1,3 @@
+envtpl==0.6.0
+circus==0.15.0
```

Comment:
@heynemann I thought we no longer need circus for thumbor 7.0 ?

also can you help me understand the difference between lib-requirements and proc-requirements ?

Reply:
I tried to keep the same stuff as there was before, didn't understand that circus was not needed anymore, we can remove proc-requirements then. Lib Requirements is there to keep the requirements file as is (didn't want to change it until we reach total parity). Just got a heads up by the mongo library maintainer that they are already using thumbor 7 in prod with their library and it's working like a charm.